### DISCUSSION TITLE
G-Helper - G and Scar 2024 - Will it have option to toggle between MultiZone and SingleZone MiniLED?

### DISCUSSION THREAD

**Discussion by xSimone** [seerge/g-helper#1918] [General]
Hello,

I want to upgrade to the Scar 18 2024 edition, I really want the MiniLED display, so I'm selling my Laptop.

Anyhow, will G-Helper have the option to toggle between Single and MultiZone for the MiniLed display? :3 

Much love, and thank you Seerge for this beautiful tool thus far, I try to save as many Asus Laptop users as I can! ;d

<img width="917" alt="12313" src="https://github.com/seerge/g-helper/assets/16178961/7a73ca63-fc95-4c88-9d9d-68afe6fafa71">

**Comment by seerge** [comment]
@xSimone hello. Most probably it will work out of the box, cause on all models Miniled switch is done in a same way (via same asus endpoint)

**Reply by xSimone** [reply]
Oooh! So exciting! <3 :3
